addons/mail/static/src/components/message/message.scss

```diff
@@ -52,21 +50,25 @@
 // ------------------------------------------------------------------
 
 .o_Message {
-    transition: background-color .5s ease-out;
+    transition: background-color .2s ease-out, opacity .5s ease-out, box-shadow .5s ease-out, transform .2s ease-out;
 
-    &.o-not-discussion.o-selected {
-        border-bottom: 1px solid darken(map-get($grays, '400'), 5%);
+    &.o-reply-highlighted {
+        transform: translateY(-#{map-get($spacers, 3)});
     }
 
-    &.o-highlighted {
-        background-color: rgba($o-brand-primary, .1);
+    .o-not-current-author {
+        opacity: .15;
+    }
```

Comment:
We did not follow our naming convention for these modifier classnames.
Since you are introducing and changing them, would camelCase e.g. `o-isReplyHighlighted` and `o-isAuthorNotCurrentUserOrGuest`